ddsrouter_yaml/test/unittest/YamlConfigurationTestUtils.hpp

```diff
@@ -154,6 +154,16 @@ void participantkind_to_yaml(
         PARTICIPANT_KIND_TAG);
 }
 
+void participantrepeater_to_yaml(
```

Comment:
This is not used. I assume it will be used in the future.

Reply:
Ok, I'd remove things that are not being currently used and implement them in the future when they are actually needed